apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V90__new_schema_rename_to_original.sql

```diff
@@ -0,0 +1,534 @@
+${logic.hsql-only}     drop constraint public.beneficiary_monthly_new_pkey;
+${logic.hsql-only} alter table public.beneficiary_monthly_new
+${logic.hsql-only}     add constraint beneficiary_monthly_pkey primary key (bene_id, year_month);
+
+alter index ${logic.psql-only} if exists 
+    public.beneficiary_monthly_new_partd_contract_year_month_bene_id_idx
+        rename to beneficiary_monthly_partd_contract_year_month_bene_id_idx;
+
+alter table if exists public.beneficiary_monthly_new
+${logic.hsql-only} add constraint beneficiary_monthly_to_benficiaries foreign key (bene_id)
+${logic.hsql-only}     references public.beneficiaries_new (bene_id);
+${logic.psql-only} rename constraint beneficiary_monthly_new_to_benficiaries
+${logic.psql-only}     to beneficiary_monthly_to_benficiaries;
+
+-- =====================
+-- skipped_rif_records
+-- =====================
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.skipped_rif_records_new_pkey
+${logic.psql-only}    rename to skipped_rif_records_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.skipped_rif_records_new
+${logic.hsql-only}     drop constraint public.skipped_rif_records_new_pkey;
+${logic.hsql-only} alter table public.skipped_rif_records_new
+${logic.hsql-only}     add constraint skipped_rif_records_pkey primary key (record_id);
+
+alter index ${logic.psql-only} if exists 
+    public.skipped_rif_records_new_bene_id_idx
+        rename to skipped_rif_records_bene_id_idx;
+
+-- =====================
+-- carrier_claims
+-- =====================
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.carrier_claims_new_pkey
+${logic.psql-only}    rename to carrier_claims_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.carrier_claims_new
+${logic.hsql-only}     drop constraint public.carrier_claims_new_pkey cascade;
+${logic.hsql-only} alter table public.carrier_claims_new
+${logic.hsql-only}     add constraint carrier_claims_pkey primary key (clm_id);
+
+alter index ${logic.psql-only} if exists 
+    public.carrier_claims_new_bene_id_idx
+        rename to carrier_claims_bene_id_idx;
+
+-- =====================
+-- carrier_claim_lines
+-- =====================
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.carrier_claim_lines_new_pkey
+${logic.psql-only}    rename to carrier_claim_lines_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.carrier_claim_lines_new
+${logic.hsql-only}     drop constraint public.carrier_claim_lines_new_pkey cascade;
+${logic.hsql-only} alter table public.carrier_claim_lines_new
+${logic.hsql-only}     add constraint carrier_claim_lines_new_pkey primary key (clm_id, line_num);
```

Comment:
this shouldnt have the _new right?

Reply:
correct...yea there was a few of them. Eyes must have been glazing over...the good/bad news is that hsql still worked, even with the bad names. Sofixed this one and the others...THANKS for catching this!